### PR TITLE
Feature/be payment route

### LINKED ISSUE #5: BE: Pay Route
POST /request/:id/pay : Pay pet sitter for this request (service completed)

### PR DESCRIPTION
**Pull request by parasVohra**
### What this PR does (required):
- Implement Route and Controller for sitters payment.
- closes #5 

### Screenshots / Videos (front-end only):
- 
Screenshot 2021-10-19 at 4 28 25 PM

### Any issues with the current functionality (optional):
- It has commits from the last PR #65. Please consider the last 2 commits for review.

### CODE REVIEW

File: server/controllers/request.js [hunk 1]
```diff
@@ -20,21 +21,33 @@ exports.requestCreate = asyncHandler(async (req, res, next) => {
...
 
   res.status(200).json({
-    message: 'request created',
+    message: "request created",
     request: request,
   });
 });
 
+exports.makePaymentIntent = asyncHandler(async (req, res, next) => {
+  console.log(process.env.STRIPE_SECRET_KEY);
```
Comment (ezeilo-su) [inline server/controllers/request.js:30]: Let's remove this comment, please.

File: server/controllers/request.js [hunk 2]
```diff
@@ -20,21 +21,33 @@ exports.requestCreate = asyncHandler(async (req, res, next) => {
...
+  const { amount } = req.body;
```
Comment (ezeilo-su) [inline server/controllers/request.js:31]: Validate this and return a `400` if not supplied.

**Review by ezeilo-su — COMMENTED** [review]
Looks good. Like I said in the comments section, you can rebase your branch to avoid this repeated showing p of old changes.
Happy coding!

**Review by moffatethan — APPROVED** [review]
Approved but keep in mind, back-end tickets shouldn't be having front-end changes

**Pull request merged**